Assets/_Application/Layers/LayerTypes/Credentials/Properties/CredentialsInputPropertySection.cs

```diff
@@ -81,12 +81,14 @@ public void ApplyCredentials()
         /// </summary>
         public void SetAuthorizationInputType(int type)
         {
-            handler.SetAuthorizationInputType((AuthorizationType)type);
+            if(handler != null)
+                handler.SetAuthorizationInputType((AuthorizationType)type);
```

Comment:
De `handler?.SetAuthorizationInputType((AuthorizationType)type);` shortcut maakt het iets minder breed met die if-statements. Dan is de if namelijk niet meer nodig. Tevens: kunnen we braces gebruiken bij alle if-statement die niet een control flow break uitvoeren? (dus: break, continue, return). Ik heb iets teveel meegemaakt dat ontwikkelaars eroverheen lazen dat dit deel van een if is, of denken iets toe te voegen aan de if maar niet door hebben dat er geen braces zijn